test/specs/platform.basic.tests.js

```diff
@@ -0,0 +1,93 @@
+import BasicChartWebWorker from 'web-worker:../BasicChartWebWorker';
```

Comment:
Is `web-worker:` a special syntax? I can't find anything about it online. I tried to check out this PR locally. It was pretty easy to rebase, but `gulp lint` is choking on this line. Not sure if I lost something while trying to rebase

Reply:
Ah, I figured out where this is coming from: https://www.npmjs.com/package/rollup-plugin-web-worker-loader

Reply:
You can stop eslint from choking on this line by changing it to:
```
import BasicChartWebWorker from 'web-worker:../BasicChartWebWorker'; // eslint-disable-line import/no-unresolved
```